sr_health_check/src/sr_health_check/sr_health_check_gui.py

```diff
@@ -167,21 +236,54 @@ def checkbox_selected(self):
         self._selected_checks['backlash'] = self._widget.checkbox_backlash.isChecked()
         self._selected_checks['overrun'] = self._widget.checkbox_overrun.isChecked()
 
+    '''
+        Initializes checks based on selected side.
+    '''
+    def side_selected(self):
+        self._side = "right" if self._widget.side_right_radio_button.isChecked() else "left"
```

Comment:
Haven't used the GUI, but from the look of it, if I uncheck the right checkbox and the left check box is empty, then it's left? Seems weird, or it deserve some explanation about the "default" behaviour

Reply:
Added a condition for left side

Reply:
Can you try something for me? What happens if you first tick `right`, then while the right is still being ticked, you tick `left`? Is that taken care of in your code? Similarly, what happens if you untick both? From what I can see here (can be wrong, though), `self._side` would remain at the same value as the last one ticked

Reply:
I think it's called auto-exclusivity in PyQt which causes it to have always 1 button in the 'clicked' state, hope this answers the question. Upon clicking one button, I initialize the checks again with class instances for the selected side.